contrib/ansible/inventory

```diff
@@ -1,10 +1,9 @@
 
 [masters]
-kube-master-test-01.example.com
+kube-master-test-[1:3].example.com
 
-[etcd]
-kube-master-test-01.example.com
+[etcd:children]
+masters
```

Comment:
Having multiple masters also seems a little premature...

Yes, lets
- remove inventory file from git
- add it to .gitignore
- create an inventory.example